### PR TITLE
chore: [CO-357] update mina-core to 2.1.6

### PR DESCRIPTION
**Pull request by frisonisland**
Updates apache mina-core to 2.1.6

### CODE REVIEW

File: store/src/java/org/apache/mina/transport/socket/nio/ZimbraSocketAcceptor.java [hunk 1]
```diff
@@ -12,219 +12,224 @@
...
+
+  @Override
+  protected void init() throws Exception {
+    selector = Selector.open();
+  }
+
+  @Override
+  protected void init(SelectorProvider selectorProvider) throws Exception {
+    selector = Selector.open();
+  }
```
Comment (Polpetta) [inline store/src/java/org/apache/mina/transport/socket/nio/ZimbraSocketAcceptor.java:80]: Sorry I don't get it why did you add this method if you don't use `selectorProvider`?

**Review by Polpetta — APPROVED** [review]
I've a question before approving

File: store/src/java/org/apache/mina/transport/socket/nio/ZimbraSocketAcceptor.java (hunk 1 above)
Comment (frisonisland) [reply store/src/java/org/apache/mina/transport/socket/nio/ZimbraSocketAcceptor.java]: @Polpetta I went for calling this.init()

**Review by keshavbhatt — APPROVED** [review]
